diff --git a/kernel/cpu_pm.c b/kernel/cpu_pm.c
index 009cc9a..461eb89 100644
--- a/kernel/cpu_pm.c
+++ b/kernel/cpu_pm.c
@@ -100,7 +100,8 @@ int cpu_pm_enter(void)
 	int nr_calls;
 	int ret = 0;
 
-	read_lock(&cpu_pm_notifier_lock);
+	//read_lock(&cpu_pm_notifier_lock);
+	__acquire(&cpu_pm_notifier_lock);
 	ret = cpu_pm_notify(CPU_PM_ENTER, -1, &nr_calls);
 	if (ret)
 		/*
@@ -108,7 +109,8 @@ int cpu_pm_enter(void)
 		 * PM entry who are notified earlier to prepare for it.
 		 */
 		cpu_pm_notify(CPU_PM_ENTER_FAILED, nr_calls - 1, NULL);
-	read_unlock(&cpu_pm_notifier_lock);
+	//read_unlock(&cpu_pm_notifier_lock);
+	__release(&cpu_pm_notifier_lock);
 
 	return ret;
 }
@@ -130,9 +132,11 @@ int cpu_pm_exit(void)
 {
 	int ret;
 
-	read_lock(&cpu_pm_notifier_lock);
+	//read_lock(&cpu_pm_notifier_lock);
+	__acquire(&cpu_pm_notifier_lock);
 	ret = cpu_pm_notify(CPU_PM_EXIT, -1, NULL);
-	read_unlock(&cpu_pm_notifier_lock);
+	__release(&cpu_pm_notifier_lock);
+	//read_unlock(&cpu_pm_notifier_lock);
 
 	return ret;
 }
