From 3263917382af02e61f12f3774c32d3324a57059f Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui at cn.fujitsu.com>
Date: Mon, 17 Aug 2015 11:48:22 +0900
Subject: [PATCH] keyutils fix x86 cflags

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e2d7e2d..f05bada 100644
--- a/Makefile
+++ b/Makefile
@@ -60,7 +60,7 @@ endif
 ifeq ($(origin USRLIBDIR),undefined)
 USRLIBDIR	:= $(patsubst /lib/%,/usr/lib/%,$(LIBDIR))
 endif
-BUILDFOR	:= $(shell file /usr/bin/make | sed -e 's!.*ELF \(32\|64\)-bit.*!\1!')-bit
+BUILDFOR	:= 32-bit
 
 LNS		:= ln -sf
 
--
1.8.4.2
