Upstream-Status: Pending

Handle NULL return from getlogin.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
diff --git a/create.c b/create.c
index e2d24d3..583af9b 100644
--- a/create.c
+++ b/create.c
@@ -77,6 +77,29 @@ static void     output(int, int *, const char *, ...) __attribute__ ((__format__
 static int      statd(FTS *, FTSENT *, uid_t *, gid_t *, mode_t *, u_long *);
 static void     statf(int, FTSENT *);
 
+char *my_getlogin()
+{
+        const char *s = getlogin();
+        if (s && *s)
+                return s;
+
+        struct passwd *p = getpwuid(geteuid());
+        char *ss;
+        if (p && p->pw_name) {
+                if (asprintf(&ss,"(no controlling terminal) %s",p->pw_name) < 0) {
+                        perror("asprintf");
+                        return NULL;
+                }
+        } else {
+                if (asprintf(&ss,"(no controlling terminal) #%d",geteuid()) < 0) {
+                        perror("asprintf");
+                        return NULL;
+            }
+        }
+
+        return ss;
+}
+
 void
 cwalk(void)
 {
@@ -92,7 +115,7 @@ cwalk(void)
                 (void)gethostname(host, sizeof(host));
                 (void)printf(
                     "#\t   user: %s\n#\tmachine: %s\n",
-                    getlogin(), host);
+                    my_getlogin(), host);
                 (void)printf(
                     "#\t   tree: %s\n#\t   date: %s",
                     fullpath, ctime(&cl));
