From 1a787af269639f2a6796ffb48467664335ddb97a Mon Sep 17 00:00:00 2001
From: OpenEmbedded <oe.patch@oe>
Date: Wed, 9 May 2018 18:25:04 +0800
Subject: [PATCH 3/9] LED: fix build error

---
 include/linux/leds.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/leds.h b/include/linux/leds.h
index ddfcb2d..271563e 100644
--- a/include/linux/leds.h
+++ b/include/linux/leds.h
@@ -52,6 +52,9 @@ struct led_classdev {
 #define LED_DEV_CAP_FLASH	(1 << 24)
 #define LED_HW_PLUGGABLE	(1 << 25)
 #define LED_PANIC_INDICATOR	(1 << 26)
+	/* Additions for Raspberry Pi PWR LED */
+#define SET_GPIO_INPUT		(1 << 30)
+#define SET_GPIO_OUTPUT		(1 << 31)
 
 	/* Set LED brightness level
 	 * Must not sleep. Use brightness_set_blocking for drivers
-- 
2.7.4

